skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -8,6 +8,12 @@
 <?php else : ?>
 	<meta name="viewport" content="width=1200">
 <?php endif ?>
+
+<? if ( Wikia::isMainPage() ): ?>
+	<? global $wgServer ?>
+	<link rel="alternate" media="only screen and (max-width: 640px)" href="<?= $wgServer ?>">
+<? endif; ?>
+
```

Comment:
It might be better to have this as part of ```$headLinks``` which adds stuff like this.  There's even an ```SEOTweaks``` extension with an ```SEOTweaksHooksHelper``` class that's already inserting a link:

https://github.com/Wikia/app/blob/HG-860/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L29